src/feature/stats/rephist.h

```diff
@@ -99,10 +99,60 @@ extern struct bw_array_t *write_array;
 #endif
 
 #ifdef REPHIST_PRIVATE
-typedef struct bw_array_t bw_array_t;
+
```

Comment:
I had to do this because bw_array_t() was private to the module
but still not accessible after #define  REPHIST_PRIVATE.
Is this ok ??

Reply:
Please put the struct in a separate file called "bw_array_st.h".
Then you can include the struct in the test file, and the code file.

That's what we do with other structs.
For examples, see:
https://github.com/torproject/tor/tree/master/src/core/or